src/UI/Common.js

```diff
@@ -8,16 +8,22 @@ const StyledModule = styled.div`
   min-height: 250px;
   display: flex;
   flex-direction: column;
-  h3 {
+  .heading {
+    font-size: 1.17em;
```

Comment:
Adding this temporarily to match the current website, which has inherited font properties from the `<h3>`.

With some styling cleanup and consistent text styles, this will get simplified.